### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by kiteloopdesign** [Solirs/gettit#3]
"installed" via tarball, on fedora 36, getting the following:

`./gettit -u 'https://www.reddit.com/r/dataisbeautiful/comments/x0fg48/ocheat_wave_the_heartbeat_and_breath_of_the_earth/' -o earth.mp4 `

[+] Initializing...

[+] Fetching url of the source video file...

[+] Downloading source video file...
2022/08/29 20:11:32 Head "": unsupported protocol scheme ""

**Comment by VakeDomen** [comment]
Same here, but built from source.

**Comment by Solirs** [comment]
I didnt expect people to still be using this haha! I'll try to reproduce the bug and fix it. Sorry for being VERY late i didn't pay much attention to this project

**Comment by Solirs** [comment]
Fixed! It turns out the requests were getting blocked by reddit. Not a problem with user agents or headers or anything, this behavior is only found for the net/http golang client (i tested with curl, python requests, and reqbin). I ended up applying a workaround i found at https://www.reddit.com/r/redditdev/comments/uncu00/go_golang_clients_getting_403_blocked_responses/ and it now works! I will pay more attention to that project from now on.
